packages/functional-tests/src/test-groups/scan-reports-test-group.ts

```diff
@@ -32,6 +32,26 @@ export class ScanReportTestGroup extends FunctionalTestGroup {
         );
     }
 
+    @test(TestEnvironment.all)
+    public async testGetConsolidatedReport(): Promise<void> {
+        const response = await this.a11yServiceClient.getScanStatus(this.testContextData.consolidatedScanId);
+        const reportsInfo = (<ScanRunResultResponse>response.body).reports;
+
+        expect((<ScanRunResultResponse>response.body).reports, 'Expected three reports to be returned').to.have.lengthOf(3);
```

Comment:
we could check for ReportFormat here like 'sarif' | 'html' | 'consolidated.html' Same can be added to testReportGenerated() in this file

Reply:
can tackle this in a separate PR!

Reply:
can address this is a separate, smaller PR